pages/docs/support/rest-api.mdx

```diff
@@ -0,0 +1,110 @@
+
+```bash
+curl -X 'GET' \
+  '{{yourinstanceurl}}/api/v1/swagger' \
+  -H 'accept: application/json'
+```
+
+with your instance's URL.
+
+## Using the Grouparoo API
+
+### Generating an API Key
+
+To get started using Grouparoo's REST API, you'll need to generate an API key. To do this, first make sure you are signed into an account with "write" access for API Keys. Then:
+
+#### UI Method
+
+1. Go to Platform > API Keys or navigate to `/apiKeys` and select "Add API Key".
+
+2. Give your API Key a name. An API Key will be generated and you will be prompted to set permissions for this API.
+
+<ImageInBrowserFrame
+  centered
+  source="Generate an API Key"
+  alt="Generate an API Key"
+  src="/images/docs/support/rest-api/apikeyconfig.png"
+  width={820}
+  height={502}
+/>
+
+To learn more about what each of the topics can do in the API, you can access the Swagger Docs for the Grouparoo API by navigating to `/swagger` in the UI of your Grouparoo Instance or going to Platform > API Documentation.
+
+#### Code Config Method
+
+You can also generate an API key object with the [`grouparoo generate` command]. To do this, run
+
+```
+grouparoo generate apiKey {{yourAPIKeyId}}
+```
+
+This command will generate a config file in a new directory `/config/apiKeys/{{yourAPIKeyId}}.js` that looks like this:
+
+```js
+exports.default = async function buildConfig() {
+  return [
+    {
+      class: "apiKey",
+      id: "{{yourAPIKeyId}}",
+      name: "{{yourAPIKeyId}}",
+      options: {
+        permissionAllRead: false, // Does this API Key have read permission to all topics?
+        permissionAllWrite: false, // Does this API Key have write permission to all topics?
+      },
+    },
+  ];
+};
+```
+
+From here, you can update the name and permissions.
+
+### Accessing the API
+
+Our Swagger docs can be found by navigating to `/swagger` in your Grouparoo Instance or by running:
+
+```bash
+curl -X 'GET' \
+  '{{yourInstanceURL}}/api/v1/swagger' \
+  -H 'accept: application/json'
+```
+
+with your instance's URL.
+
+<ImageInBrowserFrame
+  centered
+  source="Generate an API Key"
+  alt="Generate an API Key"
+  src="/images/docs/support/rest-api/swaggerdocs.png"
+  width={820}
+  height={530}
+/>
+
+To see examples of how to use your API key in the Swagger docs, click "Authorize" and enter your API Key (if pasting in your API Key, be careful about extra whitespace).
+
+<ImageInBrowserFrame
+  centered
+  source="Generate an API Key"
+  alt="Generate an API Key"
+  src="/images/docs/support/rest-api/auths.png"
+  width={820}
+  height={521}
+/>
+
+For example, if you want to see a list of all your instance's apps, you can run:
+
+```
+curl -X 'GET' \
+  '{{yourInstanceURL}}/api/v1/apps?limit=100&offset=0&apiKey={{yourAPIKey}}' \
+  -H 'accept: application/json'
+```
```

Comment:
I would love to see a workflow here - perhaps 
1. create a team
2. add someone to that team

Reply:
With that flow, it feels iffy to me that someone's password would be in their cli history in that way.  Is that being overly paranoid?  I suppose since it's HTTPS the request url itself is encrypted.

Reply:
that's a very good point.  Is there another workflow that might be good?  Maybe adding a source + property to an existing app?